build/azure-pipelines/darwin/product-build-darwin-test.yml

```diff
@@ -128,6 +127,13 @@ steps:
           yarn --cwd test/smoke compile
         displayName: Compile smoke tests
 
+      - script: |
+          set -e
+          yarn gulp \
+            compile-extension:markdown-language-features \
```

Comment:
@roblourens do you know why you need to build the markdown extension again? Does this only work if you're building that with the compiled media?

Reply:
Now that you mention it, maybe `compile-extension-media` is enough, I just pushed a change to double check. There is markdown rendering code for notebooks in this extension.